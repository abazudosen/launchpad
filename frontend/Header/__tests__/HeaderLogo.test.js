/* @flow */

import React from 'react';
import HeaderLogo from '../HeaderLogo';
import renderer from 'react-test-renderer';

test('renders normal', () => {
  const component = renderer.create(<HeaderLogo />);
  expect(component.toJSON()).toMatchSnapshot();
});
