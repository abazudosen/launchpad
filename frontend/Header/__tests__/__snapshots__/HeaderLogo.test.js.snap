// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders normal 1`] = `
<div
  className="HeaderLogo"
>
  <a
    href="https://www.apollodata.com/"
  >
    <img
      alt="Apollo Logo"
      className="Logo"
      src="icon-apollo-white.small.svg"
    />
  </a>
  <a
    href="/new"
  >
    <img
      alt="Apollo Launchpad"
      className="Subbrand"
      src="logo-apollo-white-subbrand-launchpad.svg"
    />
  </a>
</div>
`;
